README.md

```diff
@@ -152,18 +152,22 @@ The Azure MCP Server provides Azure SDK and Azure CLI developer tools. It can be
 
 ### Installation
 
-#### Getting Started Video
+#### ▶️ Getting Started Video
 
 Here's a short (16 seconds) video to help you get the Azure MCP Server installed in VS Code.
 
 https://github.com/user-attachments/assets/535f393c-0ed2-479d-9b24-5ca933293c92
 
-#### ✨ One-Click Install
+#### ✨ Quick install with VS Code
 
 Click one of these buttons to install the Azure MCP Server for VS Code or VS Code Insiders.
 
 [![Install with NPX in VS Code](https://img.shields.io/badge/VS_Code-Install_Azure_MCP_Server-0098FF?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D) [![Install with NPX in VS Code Insiders](https://img.shields.io/badge/VS_Code_Insiders-Install_Azure_MCP_Server-24bfa5?style=flat-square&logo=visualstudiocode&logoColor=white)](https://insiders.vscode.dev/redirect/mcp/install?name=Azure%20MCP%20Server&config=%7B%22command%22%3A%22npx%22%2C%22args%22%3A%5B%22-y%22%2C%22%40azure%2Fmcp%40latest%22%2C%22server%22%2C%22start%22%5D%7D&quality=insiders)
 
+You might get prompted to open the link with VS Code and then select "Install Server".
+
+![Animation](https://github.com/user-attachments/assets/2cac4c65-95ce-4945-b5dd-896ba2974dc0)
```

Comment:
I'm not sure if this gif might be too distracting, let me know your thoughts if you're against it.

Reply:
Agree, I recorded that gif to explicitly show the 3 clicks to install the server, but when I created the PR and saw it live, I don't like it anymore.

I'm going to remove it.